### PR TITLE
fix(hubble): build with node and yarn

### LINKED ISSUE #391: [Bug] mac os maven build  source code error
### Bug Type (问题类型)

others (please comment below)

### Before submit

- [X] I had searched in the [issues](https://github.com/apache/hugegraph-toolchain/issues) and found no similar issues.

### Environment (环境信息)

- Toolchain Version: v1.0.0

### Expected & Actual behavior (期望与实际表现)

 mvn clean install -DskipTests -X

hubble-dist model error
```
Caused by: org.apache.maven.plugin.MojoExecutionException: An Ant BuildException has occured: exec returned: 1
around Ant part ...<exec failonerror="true" dir="/incubator-hugegraph-toolchain/hugegraph-hubble/hubble-dist/.." executable="bash">... @ 18:147 in /incubator-hugegraph-toolchain/hugegraph-hubble/hubble-dist/target/antrun/build-main.xml

```

image

### Vertex/Edge example (问题点 / 边数据举例)

_No response_

### Schema [VertexLabel, EdgeLabel, IndexLabel] (元数据结构)

_No response_

### PR DESCRIPTION
**Pull request by liuxiaocs7**
## Purpose of the PR

- close [#391 ](https://github.com/apache/incubator-hugegraph-toolchain/issues/513)

Introduce https://github.com/eirslett/frontend-maven-plugin to install node and yarn automatically when build.

After PR: https://github.com/apache/incubator-hugegraph-toolchain/pull/542 merged and env prepared, this pr could continue.

## Verifying these changes

- [ ] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [ ] Need tests and can be verified as follows:
    - xxx

## Does this PR potentially affect the following parts?

- [ ]  Nope
- [ ]  Dependencies (add/update license info) 
- [ ]  Modify configurations
- [ ]  The public API
- [ ]  Other affects (typed here)

## Documentation Status

- [ ]  `Doc - TODO` 
- [ ]  `Doc - Done` 
- [ ]  `Doc - No Need`

### CODE REVIEW

File: hugegraph-hubble/hubble-dist/pom.xml [hunk 1]
```diff
@@ -67,6 +67,49 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>com.github.eirslett</groupId>
+                <artifactId>frontend-maven-plugin</artifactId>
+                <version>1.14.2</version>
```
Comment (imbajin) [inline hugegraph-hubble/hubble-dist/pom.xml:73]: we need update the LICENSE/NOTICE & 3rd party file maybe?
https://hugegraph.apache.org/docs/contribution-guidelines/contribute/#321-check-licenses

**Comment by liuxiaocs7** [comment]
Note: 

image

**Comment by simon824** [comment]
refer to https://stackoverflow.com/questions/60667548/frontend-maven-plugin-fails-to-execute-goal

**Review by simon824 — APPROVED** [review]

File: .github/workflows/hubble-ci.yml [hunk 2]
```diff
@@ -87,7 +69,7 @@ jobs:
         run: |
           mvn install -pl hugegraph-client,hugegraph-loader -am -Dmaven.javadoc.skip=true -DskipTests -ntp
           cd hugegraph-hubble && ls *
-          mvn -e compile -Dmaven.javadoc.skip=true -ntp
+          mvn package -DskipTests -X
```
Comment (imbajin) [inline .github/workflows/hubble-ci.yml:72]:
```suggestion
          mvn -e package -Dmaven.javadoc.skip=true -ntp
```

**Review by imbajin — DISMISSED** [review]

**Review by imbajin — APPROVED** [review]

**Review by simon824 — APPROVED** [review]

**Comment by liuxiaocs7** [comment]
Note, add `CI=false &&` will fail locally.

CC: @imbajin and @returnToInnocence

**Pull request merged**